Give more detail and context on http_addr in grafana.ini (#65187)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

### http_addr

> The host for the server to listen on. If your machine has more than one network interface, you can use this setting to expose the Grafana service only on one network interface, and not have it available on others such as the loopback interface.  If empty, will bind to all interfaces.  An empty value is equivalent to setting the value to `0.0.0.0` (all interfaces).

Reviewer: ```suggestion
The host for the server to listen on. If your machine has more than one network interface, you can use this setting to expose the Grafana service on only one network interface and not have it available on others, such as the loopback interface. An empty value is equivalent to setting the value to `0.0.0.0`, which means the Grafana service binds to all interfaces.
```

> In environments where network address translation (NAT) is being used, be careful to use the network interface address, not any final public adddress, otherwise you may see errors such as `bind: cannot assign requested address` in the logs. 

Reviewer: ```suggestion
In environments where network address translation (NAT) is used, ensure you use the network interface address and not a final public address; otherwise, you might see errors such as `bind: cannot assign requested address` in the logs. 
```

> > **Note:** It is recommended to keep this empty unless you have network security requirements which call for this configuration.

Reviewer: ```suggestion
> **Note:** It is recommended that you keep this value empty unless you have network security requirements that call for this configuration.
```

> > **Note:** It is recommended that you keep this value empty unless you have network security requirements that call for this configuration.

Reviewer 2: Can we remove this note at the end? We don't necessarily _recommend_ that it's kept empty; it's enough to provide a default. Otherwise it LGTM!
